### DISCUSSION TITLE
Is possible to select or exclude links to fetch?

### DISCUSSION THREAD

**Discussion by Iazzetta** [BeanieODM/beanie#483] [Question]
Example:

```
class ModelA(Document):
    model_b: list[Link[ModelB]]

class ModelB(Document):
    model_c: list[Link[ModelC]]

class ModelC(Document):
    model_d: Link[ModelD]
    model_e: Link[ModelE]

...
```

Using ModelA.fetch_all_links() it will return all models without DBRef. But I want exclude some models to haven't their links fetched. In my case, I want to remove fetch_links of some models inside a list. Something like that:

`ModelA.fetch_SPECIFIC_links(ModelB, ModelC, ModelE) # excluding ModelD from fetch`

I know Beanie have `fetch_link`, but I want fetch all except a few links of my documents. It's possible?

**Comment by roman-right** [comment]
It is not possible currently, but this sounds reasonable. I'll add this feature. Thank you
